index.js

```diff
@@ -224,7 +225,12 @@ class K8sVMExecutor extends Executor {
 
         const podConfig = yaml.safeLoad(podTemplate);
 
-        setNodeSelector(podConfig, this.nodeSelectors);
+        const diskConfig = annotations[DISK_RESOURCE] || '';
+        const nodeSelectors = diskConfig.toUpperCase() === 'HIGH' ? { disk: 'high' } : {};
+
+        hoek.merge(nodeSelectors, this.nodeSelectors);
```

Comment:
`this.nodeSelectors` will override `nodeSelectors` if there's any conflicts. Is this a concern at all?

Reply:
I believe it should be fine since we currently do not specify a default selector in regards to any resource.